doc/content/getting-started/installation/certificates.md

```diff
@@ -120,3 +122,5 @@ config/
 └── stack/
     └── ttn-lw-stack-docker.yml    # configuration file for {{% tts %}}
 ```
+
+Be sure to configure `docker-compose.yml` and `ttn-lw-stack-docker.yml` for your custom certificates, as shown in [using custom certificates](#using-custom-certificates).
```

Comment:
**Using custom certificates** section implies that the user should uncomment lines for ACME in `ttn-lw-stack-docker.yml`, this might be confusing I think. Maybe add a sentence that the lines with `file` source should be uncommented instead